src/unit/get.js

```diff
@@ -0,0 +1,82 @@
+define([
+	"./categories"
+], function( unitCategories ) {
+
+function stripPluralGarbage( data ) {
+	var aux, pluralCount;
+
+	if ( data ) {
+		aux = {};
+		for ( pluralCount in data ) {
+			aux[ pluralCount.replace( /unitPattern-count-/, "" ) ] = data[ pluralCount ];
+		}
+	}
+
+	return aux;
+}
+
+/**
+ * get( unit, form, cldr )
+ *
+ * @unit [String] The full type-unit name (eg. duration-second), or the short unit name
+ * (eg. second).
+ *
+ * @form [String] A string describing the form of the unit representation (eg. long,
+ * short, narrow).
+ *
+ * @cldr [Cldr instance].
+ *
+ * Return the plural map of a unit, eg: "second"
+ * { "one": "{0} second",
+ *   "other": "{0} seconds" }
+ * }
+ *
+ * Or the Array of plural maps of a compound-unit, eg: "foot-per-second"
+ * [ { "one": "{0} foot",
+ *     "other": "{0} feet" },
+ *   { "one": "{0} second",
+ *     "other": "{0} seconds" } ]
+ *
+ * Or undefined in case the unit (or a unit of the compound-unit) doesn't exist.
+ */
+var get = function( unit, form, cldr ) {
+	var ret;
+
+	// Ensure that we get the 'precomputed' form, if present.
```

Comment:
Let's include we're treating `/` like `-per`, something like "Treat `/` like `-per-` and ensure that we get the 'precomputed' form, if present."